runtime/shibuya/src/lib.rs

```diff
@@ -648,7 +648,11 @@ impl pallet_contracts::Config for Runtime {
     type CallStack = [pallet_contracts::Frame<Self>; 31];
     type WeightPrice = pallet_transaction_payment::Pallet<Self>;
     type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
-    type ChainExtension = (DappsStakingExtension<Self>, XvmExtension<Self>);
+    type ChainExtension = (
```

Comment:
Whenever a change is introduced which modifies the runtime binary, we bump up the `spec_version`.

This is needed for `astar-runtime`, `shibuya-runtime` and `shiden-runtime`. `local-runtime` can be ignored since it's not a live chain (we always keep it at `1`).

Unlike bumping the version in the `.toml` file where we **always** bump up all the files, for `spec_version` we only do it for the impacted files. In your case, it's just `shibuya`.